docs/series/README.md

```diff
@@ -1,6 +1,52 @@
----
-title: "Howto Series"
----
+# Howto Series: AlmaLinux Tutorials 🚀
 
-# Series Page
+Welcome to our Howto series on AlmaLinux! In this series, we provide comprehensive guides and tutorials to help you learn and master various aspects of AlmaLinux. Whether you are interested in Nginx or system-related topics, we've got you covered.
 
+## Nginx Series
+
+The Nginx series focuses on all things related to Nginx web server. From installation to configuration, security, and optimization, you will find step-by-step guides to help you deploy and manage a powerful web server on AlmaLinux.
+
+Browse the articles in the Nginx series:
+
+- [A01](nginx/NginxSeriesA01.md) ❯ Beginner's Guide
+- [A02 R8](nginx/NginxSeriesA02R8.md) ❯ AlmaLinux 8.x Installation Examples
+- [A02 R9.1](nginx/NginxSeriesA02R91.md) ❯ AlmaLinux 9.1 Installation Examples
+- [A02 R9.2](nginx/NginxSeriesA02R92.md) ❯ AlmaLinux 9.2 Installation Examples
+- [A03](nginx/NginxSeriesA03.md) ❯ Default Configuration Guide
+- [A04](nginx/NginxSeriesA04P1.md) ❯ Secure Nginx Deployment
+
+## System Series
+
+**The vision:** (In near future) The System series covers a wide range of system-related topics on AlmaLinux. This guides will help you enhance your system administration skills and ensure the stability and security of your AlmaLinux environment.
+
+Explore the articles in the System series:
+
+- [A01](system/SystemSeriesA01.md) ❯ Application Streams
+- [A02](system/SystemSeriesA02.md) ❯ Firewalld: A Beginner's Guide
+- [A03](system/SystemSeriesA03.md) ❯ NVIDIA Driver Installation Guide
+- [A03 R8](system/SystemSeriesA03R8.md) ❯ NVIDIA Driver: Installation on 8.x
+- [A03 R9](system/SystemSeriesA03R9.md) ❯ NVIDIA Driver: Installation on 9.x
+- [A04](system/SystemSeriesA04.md) ❯ Snapd Installation Guide
+- [A05](system/SystemSeriesA05.md) ❯ Multimedia Codecs Installation Guide
+
+Dive into our Howto series and unlock the full potential of AlmaLinux. Happy hacking! 🐧💻🚀
+
+----
+
+##### Trademark Notices
+:::warning
+
+**AlmaLinux** is a registered trademark of AlmaLinux OS Foundation.
+
+**Linux®** is a registered trademark of Linus Torvalds in the U.S. and other countries.
+
+**macOS** are trademarks of Apple Inc., registered in the U.S. and other countries.
+
+**NVIDIA** is a registered trademark of NVIDIA Corporation in the U.S. and other countries.
+
+**Red Hat** is registered trademark of Red Hat, Inc. or its subsidiaries in the United States and other countries.
```

Comment:
**Red Hat** is a registered trademark of Red Hat, Inc. or its subsidiaries in the United States and other countries.